components/organisms/ContributorProfileTab/contributor-profile-tab.tsx

```diff
@@ -74,23 +121,24 @@ const ContributorProfileTab = ({
     }
   }, [highlights]);
 
+  useEffect(() => {
+    if (userInterests && userInterests.length > 0) setSelectedInterest(userInterests.split(","));
+
+    getRepoFullNameByInterests();
+  }, [userInterests]);
+
   return (
     <Tabs defaultValue={uppercaseFirst(currentPathname as string)} className="">
       <TabsList className="justify-start w-full border-b">
-        <TabsTrigger
-          className="data-[state=active]:border-sauced-orange data-[state=active]:border-b-2 text-2xl"
-          value="Highlights"
-          onClick={() => handleTabUrl("Highlights")}
-        >
-          Highlights
-        </TabsTrigger>
-        <TabsTrigger
-          className="data-[state=active]:border-sauced-orange data-[state=active]:border-b-2 text-2xl"
-          value="Contributions"
-          onClick={() => handleTabUrl("Contributions")}
-        >
-          Contributions
-        </TabsTrigger>
+        {tabLinks.map((tab) => (
+          <TabsTrigger
+            key={tab}
+            className="data-[state=active]:border-sauced-orange data-[state=active]:border-b-2 text-2xl data-[state=active]:font-bold data-[state=active]:text-transparent data-[state=active]:bg-clip-text data-[state=active]:bg-gradient-to-r data-[state=active]:from-[#EA4600] data-[state=active]:to-[#EB9B00]"
```

Comment:
This design was misinterpreted. Only Recommendations should have the gradient. This is meant to be a special tab and entice users without highlights and contributions to click and see.

<img width="304" alt="Screen Shot 2023-05-15 at 9 18 52 AM" src="https://github.com/open-sauced/insights/assets/5713670/8b267ba3-af2c-402f-b461-241e82fe0d25">

Reply:
Ok...

Reply:
reversed 👍